content/posts/formularios-para-sites-estaticos.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Formsquash, formulários para sites estáticos
+authors: Jean Lucas de Carvalho
+type: post
+image: https://formsquash.io/static/images/cover.jpg
+date: 2018-05-24
+excerpt: Autenticação de usuário com o Firebase Auth
+categories:
+  - Artigos
+  - Ferramentas
+  - HTML
+  - Tecnologia e Tendências
+tags:
+  - formsquash
+  - html
+  - websites
+---
+
+# Formsquash, formulários para sites estáticos
```

Comment:
Esse título não é necessário, pois já está aqui title: Formsquash, formulários para sites estáticos

Reply:
Força do hábito ;)